styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -92,6 +94,27 @@ public String createExecution(RunState runState, final String execName, final Fl
     }
   }
 
+  @Override
+  public void terminateExecution(RunState runState, FlyteExecutionId flyteExecutionId) {
+    requireNonNull(runState, "runState");
+    requireNonNull(flyteExecutionId, "flyteExecutionId");
+    try {
+      flyteAdminClient.terminateExecution(
```

Comment:
what happens if the workflow is already terminated?

Reply:
It does nothing, at least in the cli. I can create a small program to validate what happens with the API and either I will make changes or add a comment describing that nothing happens if the workflow execution have been terminated

Reply:
sounds good. i think it's good to know what happens just to make sure that we don't mistreat something

Reply:
I confirmed that using the the grpc client we gets the same result: it is ok to terminate a workflow more than once without throwing an execption. I will add a comment to make it clear